frontend/www/js/omegaup/components/admin/Support.test.ts

```diff
@@ -0,0 +1,23 @@
+import { shallowMount } from '@vue/test-utils';
+import T from '../../lang';
+
+import admin_Support from './Support.vue';
+
+describe('Support.vue', () => {
+  it('Should handle support page for an specific user', () => {
```

Comment:
nit: este usuario no es realmente específico

```suggestion
  it('Should handle support page for users without a birth date', () => {
```